redash/app.py

```diff
@@ -40,6 +46,10 @@ def create_app():
     sentry.init()
     app = Redash()
 
+    # Initialize the Flask-Babel extension
+    babel = Babel(app)
+    babel.locale_selector_func = get_locale
```

Comment:
Why did you decide to use this way instead of like the documentation in the class initialization?


```suggestion
    babel = Babel(app, locale_selector=get_locale)
```

Reply:
yeah it was a mistake, i fixed it here in this commit https://github.com/metr-systems/redash/pull/19/commits/d486655258eb0d7c1795ce409df033c1891d3c84 , thanks